SQL/Archive/17.1/2017-03-24-examiner_decoupling_from_site.sql

```diff
@@ -0,0 +1,40 @@
+DROP TABLE IF EXISTS `examiners_psc_rel`;
+CREATE TABLE `examiners_psc_rel` (
+  `examinerID` int(10) unsigned NOT NULL,
+  `centerID` tinyint(2) unsigned NOT NULL,
+  `active` enum('Y','N') NOT NULL DEFAULT 'Y',
+  `pending_approval` enum('Y','N') NOT NULL DEFAULT 'N',
+  PRIMARY KEY  (`examinerID`,`centerID`),
+  KEY `FK_examiners_psc_rel_2` (`centerID`),
+  CONSTRAINT `FK_examiners_psc_rel_1` FOREIGN KEY (`examinerID`) REFERENCES `examiners` (`examinerID`) ON DELETE CASCADE ON UPDATE CASCADE,
+  CONSTRAINT `FK_examiners_psc_rel_2` FOREIGN KEY (`centerID`) REFERENCES `psc` (`CenterID`) ON DELETE CASCADE ON UPDATE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+SELECT "Creating copy of `examiners` table as `examiners_17_0`";
+CREATE TABLE examiners_17_0 LIKE examiners;
+INSERT examiners_17_0 SELECT * FROM examiners;
+
+-- CLEAN-UP
+UPDATE examiners SET full_name = TRIM(full_name);
```

Comment:
I have looked at a couple instruments, they seems to hold the ExaminerID. They still need to be updated befor removing duplicate in th eexaminer table.